.circleci/config.yml

```diff
@@ -261,6 +266,60 @@ jobs:
               export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
               . activate habitat; cd habitat-sim
               python examples/example.py --scene data/scene_datasets/habitat-test-scenes/van-gogh-room.glb --silent --test_fps_regression $FPS_THRESHOLD
+      - add_ssh_keys:
+          fingerprints:
+            - "18:88:e0:37:b1:b3:7a:23:aa:1e:f7:43:a8:5b:8e:05"
+      - run:
+          name: Update documentation
+          command: |
+              export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
+              . activate habitat;
+              cd habitat-sim
+              # Rebuild with all options enabled
+              ./build.sh --with-cuda --with-bullet
+              cd docs
+              conda install -y -c conda-forge doxygen
+              conda install -y  jinja2 pygments docutils
+              conda install -y -c conda-forge pelican
+              mkdir -p ../build/docs
+              git submodule update --init
+              doxygen
+              ./build-public.sh
+              cd ../..
+
+              # Update website
+              git clone git@github.com:facebookmicrosites/habitat-website.git
+              cd habitat-website
+              git submodule update --init
+
+              for dir in habitat-sim habitat-cpp
+              do
+                  rm -rf published/docs/${dir}
+                  cp -r ../habitat-sim/build/docs-public/${dir} published/docs/.
+              done
+              git checkout published/docs/habitat-sim/index.html
+
+              git config --global user.name "Habitat"
+              git config --global user.email habitat@fb.com
+              NOW=$(date +"%m-%d-%Y")
+              git add .
+              git commit -m "Build ${NOW}"
+              git status
+              git stash
+              git push origin master
```

Comment:
It might make sense to push both master/gh-pages together once all of the required copy/pasting is finished. If there is some issue in the below commands, master/gh-pages won't go out of sync.